setup.cfg

```diff
@@ -1,26 +1,56 @@
 [metadata]
 name = grizli
-description = Grism analysis pipeline
+version = 1.6.0
 author = G. Brammer
 author_email = gbrammer@gmail.com
+description = Grism redshift and line analysis software
+long_description = file: README.rst
+url = https://github.com/gbrammer/grizli
 license = MIT
-url = http://github.com/gbrammer/grizli
-edit_on_github = False
-github_project = gbrammer/grizli
+license_files = LICENSE.txt
+project_urls = 
+    Documentation = https://grizli.readthedocs.io/
+    Source = https://github.com/astropy/astropy
+classifiers=
+        Development Status :: 1 - Planning
```

Comment:
This is probably not a good classifier for the code.

Reply:
Maybe we can bump up to `Development Status :: 3 - Alpha`?